packages/edit-site/src/components/page-patterns/index.js

```diff
@@ -187,11 +195,14 @@ function Author( { item, viewType } ) {
 
 function Title( { item } ) {
 	const isUserPattern = item.type === PATTERN_TYPES.user;
+	const isTemplatePart = item.type === TEMPLATE_PART_POST_TYPE;
 	const { onClick } = useLink( {
 		postType: item.type,
-		postId: isUserPattern ? item.id : item.name,
+		postId: isUserPattern || isTemplatePart ? item.id : item.name,
 		canvas: 'edit',
 	} );
+	const title =
+		typeof item.title === 'string' ? item.title : item.title.rendered;
```

Comment:
Should this also use `defaultGetTitle`?

Reply:
Also, do we still need `decodeEntities` here?

Reply:
defaultGetTitle is in another package, I don't want to export a small util like that.

Reply:
actually, no it's in the same package but I'm still choosing to not using for now, cause it's "search" specific for now and this would break that.